test/e2e/scale_test.go

```diff
@@ -24,21 +24,23 @@ import (
 	"k8s.io/utils/pointer"
 )
 
-var _ = Describe("When scale testing using in-memory provider [Scale]", func() {
+var _ = Describe("When scale testing using in-memory provider with upgrades [Scale]", func() {
```

Comment:
AFAIK this test is largely just representational - so I enabled all the features.